app/views/errors/not_found.html.erb

```diff
@@ -2,18 +2,18 @@
 
 <div class="container text-center">
   <div class="row row-errorpage">
-    <div class="col-xs col-sm col-md col-lg">
+    <div class="col col-sm col-md col-lg">
```

Comment:
there’s a chance doing this could impact the layout at specific breakpoints. Since this file is part of the error handling UI, which needs to be consistent across all screen sizes, I think it’s safer to keep the explicit breakpoint classes for now.